src/components/Hero.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+export const Hero = () => {
+  return (
+    <>
+      <div className="hero-section">
+        <div className="overlay">
+          <h1>GET FIT</h1>
+          <p>Your one stop fitness solution</p>
+          <a href="#category" className="pt-1">
```

Comment:
Here, instead of using `anchor` tag,  use `Link` tag.

Reply:
Thanks for the suggestion. After I create product listing page I'll change this to `Link` tag.